nbstripout/_nbstripout.py

```diff
@@ -483,14 +502,17 @@ def main():
     input_stream = io.TextIOWrapper(sys.stdin.buffer, encoding='utf-8') if sys.stdin else None
     output_stream = io.TextIOWrapper(sys.stdout.buffer, encoding='utf-8', newline='')
 
+    any_change = False
     for filename in args.files:
         if not (args.force or filename.endswith('.ipynb') or filename.endswith('.zpln')):
             continue
 
         try:
             with io.open(filename, 'r+', encoding='utf8', newline='') as f:
                 out = output_stream if args.textconv or args.dry_run else f
-                process_notebook(f, out, args, extra_keys, filename)
+                any_local_change = process_notebook(f, out, args, extra_keys, filename)
+                any_change = any_change or any_local_change
```

Comment:
```suggestion
                if process_notebook(f, out, args, extra_keys, filename):
                    any_change = True
```
Wouldn't this be easier?

Reply:
It felt easier to read at the time (I am accepting the suggestion)